docs/source/support/overview.md

```diff
@@ -0,0 +1,14 @@
+# Support overview
```

Comment:
Instead of having a dedicated overview page, I'd move it to support/index - this way there isn't just a page with bullet points but some actual content there and the navigation seems more natural.

Reply:
this is on purpose, if you are using this top menu to navigate, you want to have all the sublinks when you click on "SUPPORT"
![Screenshot from 2023-10-25 09-10-07](https://github.com/scylladb/scylla-operator/assets/11869554/fde77bf7-c4df-4c2e-8af4-f11af11abb9f)

Reply:
so it think there should be a dedicated `Overview page` with some intro but the "index" should have the TOC which has to have a ref to every subpage. it might be nice to enrich it, like say [Kube does ](https://kubernetes.io/docs/concepts/) but I am not sure we have a generated way to do that - do we?

Reply:
I'm not aware of a generated way, the toctree allows you to change the text of the hyperlink but I guess that's about it. Your best bet is to do it the way we do in the other indexes - hide the generated tree and write your own (e.g. https://github.com/scylladb/scylla-operator/blob/master/docs/source/multidc/index.rst)

Reply:
https://github.com/scylladb/sphinx-scylladb-theme/issues/935

Reply:
> indexes is required to list all the pages and this would get out of hand if written manually

You still have to add it manually though? I don't see any benefit of only adding an entry to the index over adding it to the hidden index and also adding a manual bullet point.

Reply:
using toc makes it match the menu on the left, always. To my knowledge those items are generated from the same list and that's what creates the hierarchy. If you hide the menu and write it by hand, pages may be left out and some will be reachable only through the menu and not the index page.

Reply:
Ok, let's just choose an approach and try to make it concise across the entire documentation in the long run. I'm ok with leaving it as is and going back to it once the above issue is resolved.

Reply:
Jfyi though, while I agree with you on the benefits of using the generator toc, it seems like ScyllaDB OS docs hide it as well:
https://raw.githubusercontent.com/scylladb/scylladb/scylla-5.2.0/docs/using-scylla/index.rst
![image](https://github.com/scylladb/scylla-operator/assets/32443348/410e3c6e-0e33-4a8b-8f9b-5c3602c5a0e7)